eessi/testsuite/tests/apps/gromacs.py

```diff
@@ -91,3 +91,13 @@ def set_omp_num_threads(self):
 
         self.env_vars['OMP_NUM_THREADS'] = omp_num_threads
         log(f'env_vars set to {self.env_vars}')
+
+    @run_after('setup')
+    def set_up_offline_run(self):
+        """
+        Set environments variables to run offline or skip the test
+        """
+        if 'internet_access' in self.current_partition.extras:
+            if self.current_partition.extras['internet_access'] == 'offline':
+                msg = f'Test for {self.module_name} cannot be run on offline machine'
+                self.skip_if(self.module_name == self.module_name, msg)
```

Comment:
let's move this to `eessi_mixin.py`, and add an attribute, say `requires_internet_access`, to these 3 tests that need it?